mars/metric/backends/console/tests/pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
```

Comment:
Remove this file as we config pytest via commandline arguments in CI.

Reply:
Ok.